src/model/data/fridge/REST.yaml

```diff
@@ -159,7 +165,30 @@ paths:
         '500':
           description: Unexpected error prevented server from fulfilling request.
 
-  /v1/fridges/{id}/reports:
+  /reports/last:
```

Comment:
Sounds good, make a ticket for it: https://trello.com/c/hdvp8cEX/248-cfm219-add-reports-last-api